docker-hub/billing/faq.md

```diff
@@ -1,211 +1,83 @@
 ---
-description: Pricing & Pull Rate Limiting FAQs
-keywords: Docker, Docker Hub, pricing FAQs, pull rate limiting FAQs, subscription, platform
-title: Pricing & Pull Rate Limiting FAQs
+description: Subscription & Billing FAQs
+keywords: Docker, Docker Hub, billing FAQs, pull rate limiting FAQs, subscription, platform
+title: Subscription & Billing FAQs
 ---
 
-## Pricing FAQs
+## Subscription Tiers
 
-As of May 14, 2020, Docker announced [new pricing changes](/docker-hub/billing/) for Docker Hub subscriptions.
+### What subscriptions levels are available for Docker?
 
-### What plans and pricing changes did Docker announce on May 14?
+[Docker Subscriptions](https://www.docker.com/pricing/resource-consumption-updates) come in three levels:
```

Comment:
Could you add {:target="_blank" rel="noopener" class="_"} so it opens the link in a new tab?